packages/@aws-cdk/aws-glue/lib/table.ts

```diff
@@ -98,7 +153,7 @@ export class Table extends cdk.Construct implements ITable {
   }
 
   public readonly database: IDatabase;
-  public readonly bucket: s3.Bucket;
+  public readonly bucket: s3.IBucket;
   public readonly prefix: string;
```

Comment:
"prefix" might be a bit ambiguous, maybe `s3Prefix` or `s3KeyPrefix`?

Reply:
Agreed. Changed to `s3Prefix`.